crates/byte/src/lib.rs

```diff
@@ -1,34 +1,87 @@
 // SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
 // SPDX-License-Identifier: Apache-2.0
 
-use std::marker::PhantomData;
-use std::ops::{Deref, DerefMut};
-use std::str::FromStr;
-
-use serde::{de::Error as _, Deserialize, Serialize};
-
-pub trait Config {
-    const CONFIG: base64::Config;
+//! This crate provides a [`Bytes`] type which wraps most types that represent
+//! a contiguous array of bytes. It provides implementations for easy
+//! conversions to and from Base64 representations in string contexts.
+
+#![no_std]
```

Comment:
Why the change to `no_std`?

Reply:
@rjzak Because this is a library. If you're writing a library that doesn't explicitly need `std`, then it is a good idea to make it `no_std`. This way other people can use it in `no_std` contexts.

Reply:
Yes. I'll do that in the future.